Makefile

```diff
@@ -2,12 +2,12 @@
 .PHONY : test
 
 update-dependencies:
-	docker pull jwilder/docker-gen:latest
-	docker pull nginx:latest
+	docker pull jwilder/docker-gen:0.7.3
+	docker pull nginx:1.9.12
```

Comment:
IMO, latest requires testing each built image before deploy and not many of our apps do so (yet).